include/openthread/link_metrics.h

```diff
@@ -115,6 +138,46 @@ otError otLinkMetricsQuery(otInstance *                aInstance,
                            otLinkMetricsReportCallback aCallback,
                            void *                      aCallbackContext);
 
+/**
+ * This function sends an MLE Link Metrics Management Request to configure a Forward Tracking Series.
+ *
+ * @param[in] aInstance          A pointer to an OpenThread instance.
+ * @param[in] aDestination       A pointer to the destination address.
+ * @param[in] aSeriesId          The ID of the Series to operate with.
+ * @param[in] aSeriesFlags       A pointer to the Series Flags that specifies which frames are to be accounted.
```

Comment:
It shouldn't be `null`. I think it would be difficult for users to use using `uint8_t` here (That was Abtin's suggestion in the previous Link Metrics PR) because users need to play with the bits then. I think `aSeriesFlags` is just similar with `aLinkMetricsFlags` and we can use it in the same way.

Reply:
Add note that the pointer could not be null or maybe simply pass the object `otLinkMetricsSeriesFlags  aSeriesFlags` other than a pointer?

Reply:
I would prefer to add some note here because all arguments for structs in public API are passed by pointer, I think it would be better to keep consistent.